package.json

```diff
@@ -96,11 +95,14 @@
   "lint-staged": {
     "*": [
       "prettier --write"
+    ],
+    "*.{ts,js}": [
+      "tslint"
```

Comment:
I think it's a bad idea. What I mean:
When husky executes, it's looking for errors, prettier fixes errors, but tslint doesn't; it's just pointing at the error.
So it's breaking the execution and reverts changes.

For example:
There is no semi-column
![image](https://user-images.githubusercontent.com/43093994/120286531-05694e80-c2c7-11eb-9565-390213e19fff.png)

In our previous version without tslint, it will add it and commit. With your updates, it won't fix that and commit without it.

Result of new changes:
Found errors
![image](https://user-images.githubusercontent.com/43093994/120286991-7c064c00-c2c7-11eb-9b76-bd9af8fe7853.png)

Prettier reverted changes and didn't fix semi-column:
![image](https://user-images.githubusercontent.com/43093994/120287061-8de7ef00-c2c7-11eb-8302-85f084e9863b.png)

Commit was made:
![image](https://user-images.githubusercontent.com/43093994/120287104-9a6c4780-c2c7-11eb-9517-1ab3e122a4f8.png)

Previous:
No semi-column
![image](https://user-images.githubusercontent.com/43093994/120287302-cee00380-c2c7-11eb-97ac-1efc2f29efe3.png)

Fixed semi-columns:
![image](https://user-images.githubusercontent.com/43093994/120287684-25e5d880-c2c8-11eb-9327-82153794c108.png)

Commit was successful:
![image](https://user-images.githubusercontent.com/43093994/120287734-3433f480-c2c8-11eb-90a8-1dca7a5c73e2.png)